src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
+    this.info = data
+    this.priority = priority
+    this.before = proceeds
+  }
+}
+
+class priorityQueue {
+  constructor() {
+    this.head = null
+    this.tail = null
+    this.count = 0
+  }
+}
+
+  enqueue(value, num) {// adds an element with priority (number) to the back of the queue.
```

Comment:
Comments will be removed, just using for reference from original specs of project